easybuild/easyblocks/p/pytorch.py

```diff
@@ -253,7 +255,31 @@ def test_step(self):
             'python': self.python_cmd,
             'excluded_tests': ' '.join(excluded_tests)
         })
-        super(EB_PyTorch, self).test_step()
+
+        (out, ec) = super(EB_PyTorch, self).test_step(return_output_ec=True)
+        if ec:
+            print_warning("Test command had non-zero exit code (%s)!" % ec)
+
+        ran_tests_regex = re.compile(r"^Ran (?P<test_cnt>[0-9]+) tests in", re.M)
+        ran_tests_hits = ran_tests_regex.findall(out)
```

Comment:
Why the explicit compile?
```suggestion
        ran_tests_hits = re.findall(r"^Ran (?P<test_cnt>[0-9]+) tests in", out, re.M)
```